src/backend/gporca/libgpopt/src/translate/CTranslatorExprToDXL.cpp

```diff
@@ -3902,16 +3902,18 @@ CTranslatorExprToDXL::PdxlnCorrelatedNLJoin(
 	COperator::EOperatorId op_id = pexpr->Pop()->Eopid();
 	CDXLNode *pdxlnCond = nullptr;
 
-	// Create a subplan with a Boolean from the inner child if we have a Const True as a join condition.
-	// One scenario for this is when IN sublinks contain a projection from the outer table only such as:
-	// select * from foo where foo.a in (select foo.b from bar);
-	// If bar is a very small table, ORCA generates a CorrelatedInLeftSemiNLJoin with a Const true join filter
-	// and condition foo.a = foo.b is added as a filter on the table scan of foo. If bar is a large table,
-	// ORCA generates a plan with CorrelatedInnerNLJoin with a Const true join filter and a LIMIT over the
-	// scan of bar. The same foo.a = foo.b condition is also added as a filter on the table scan of foo.
+	// Create a subplan with a Boolean from the inner child if we have a Const
+	// True as a join condition.
+	// One scenario for this is when sublink contain a projection from the outer
```

Comment:
Do we really need a new paragraph here?
```suggestion
	// Create a subplan with a Boolean from the inner child if we have a Const
	// True as a join condition. One scenario for this is when sublink contain a
```